components/style/values/computed/length.rs

```diff
@@ -64,26 +63,16 @@ impl ToComputedValue for specified::Length {
     }
 }
 
-#[derive(Clone, Copy, Debug, PartialEq)]
-#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
 #[allow(missing_docs)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+#[derive(Clone, Copy, Debug, PartialEq, ToAnimatedZero)]
 pub struct CalcLengthOrPercentage {
+    #[animation(equal)]
```

Comment:
The semantics of this on `ToAnimatedZero` are quite different to `Animate`... I'm not sure how I feel about this.